lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -265,7 +265,7 @@ sub start
         send_federation       => ( not $self->{workers} ),
         enable_media_repo     => ( not $self->{workers} ),
         run_background_tasks_on  => ( $self->{workers} ? "background_worker1" : "master" ),
-        worker_to_update_user_directory  => ( $self->{workers} ? "user_dir" : "null" ),
+        worker_to_update_user_directory  => ( $self->{workers} ? "user_dir" : undef ),
```

Comment:
wouldn't it be better not to set it at all, so that synapse uses its default?

```suggestion
        $self->{workers} ? (
            worker_to_update_user_directory  => "user_dir",
        ) : (),
```

Reply:
Yes, this'd be better. Thanks, i didn't know this was possible in perl.